packer/linux/scripts/install-docker.sh

```diff
@@ -30,18 +31,18 @@ sudo curl -Lfs -o /etc/systemd/system/docker.socket https://raw.githubuserconten
 sudo systemctl daemon-reload
 sudo systemctl enable docker.service
 
-echo "Downloading docker-compose..."
-sudo curl -Lsf -o /usr/bin/docker-compose https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
-sudo chmod +x /usr/bin/docker-compose
-docker-compose --version
+if [ ${MACHINE} == "x86_64" ]; then
+	echo "Downloading docker-compose..."
+	sudo curl -Lsf -o /usr/bin/docker-compose https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
+	sudo chmod +x /usr/bin/docker-compose
+	docker-compose --version
+fi
```

Comment:
```suggestion
else
  sudo pip3 install docker-compose
fi
```

Wouldn't this be enough to install docker-compose directly on arm?